pkg/syncer/metadatasyncer.go

```diff
@@ -2321,6 +2324,8 @@ func csiPVUpdated(ctx context.Context, newPv *v1.PersistentVolume, oldPv *v1.Per
 					"Calling CreateVolume with BackingID to mark volume as Container Volume.", oldPv.Spec.CSI.VolumeHandle)
 				// Call CreateVolume for Static Volume Provisioning.
 				_ = createCnsVolume(ctx, oldPv, metadataSyncer, cnsVolumeMgr, volumeType, vcHost, metadataList, volumeHandle)
+				errMsg := fmt.Sprintf("Failed to create volume on VC %s", vcHost)
```

Comment:
I think we need to handle some failure before adding event on the PV. why are we generating error event right after `createCnsVolume` without checking error returned  from `createCnsVolume`

Reply:
We are not printing error over here because it is being handled in the function itself. So we do not need to log the error again.